### PR TITLE
Update _help.py to include --version

### PR DESCRIPTION
**Pull request by larryms**
include documentation of '--version' option.  
Solves Issue #6096 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6259
This is an **experimental preview** for @microsoft users.

File: src/azure-cli-core/azure/cli/core/_help.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 Welcome to the cool new Azure CLI!
 
+Use `az --version` to display the current version of the cli and modules.
```
Comment (derekbekoe) [inline src/azure-cli-core/azure/cli/core/_help.py]: Can you remove ` of the cli and modules`?

Comment (larryms) [reply src/azure-cli-core/azure/cli/core/_help.py]: @derekbekoe I've updated per your requet

**Comment by larryms** [comment]
@derekbekoe updated text per your request

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6259
This is an **experimental preview** for @microsoft users.

**Comment by derekbekoe** [comment]
Re-opening to trigger CI against latest branch

**Comment by derekbekoe** [comment]
Thanks for the PR @larryms!

**Pull request merged**